meson.build

```diff
@@ -2,7 +2,7 @@ project('libratbag', 'c', 'cpp',
 	version : '0.9.903',
 	license : 'MIT/Expat',
 	default_options : [ 'c_std=gnu99', 'warning_level=2' ],
-	meson_version : '>= 0.40.0')
+	meson_version : '>= 0.48.0')
```

Comment:
that's going to break a few things. meson 0.48 was only released 2 months ago and [isn't shipped in Ubuntu](https://packages.ubuntu.com/search?keywords=meson). given how many issues we already have with Ubuntu I'd rather not add to them...